CMakeLists.txt

```diff
@@ -46,6 +46,8 @@ option(STABLEHLO_BUILD_EMBEDDED "Build StableHLO as part of another project" OFF
 option(STABLEHLO_ENABLE_BINDINGS_PYTHON "Enables StableHLO Python bindings" OFF)
 option(STABLEHLO_ENABLE_STRICT_BUILD "Build StableHLO with strict warnings and warnings as errors" OFF)
 option(STABLEHLO_ENABLE_SANITIZER "Enable a sanitizer [OFF, address]" OFF)
+option(STABLEHLO_ENABLE_SPLIT_DWARF "Enable split DWARF if the platform supports it" OFF)
```

Comment:
Should this default to ON? Are there any downsides to using this?

Reply:
Maybe -- It might be good to get a place (maybe with Nix) where everything is controlled.
Right now I'm finding that on my machine I build with GCC but the CI uses Clang.
Removing all the variability and forcing a consistent toolchain would let us remove a lot of these options and make them default ON